9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml

```diff
@@ -29,7 +29,7 @@
         <role_conditions/>
         <script><![CDATA[(function executeRule(current, previous /*null when async*/) {
 
-	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+\+)/g);
+	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+){2}/gm);
```

Comment:
ok, so, uh, my bad... this regex doesn't work with emca. It needs to be
`/(?:<@)(.*?)(?=>(?:\s*\+){2})/gm`